### PR TITLE
Enforce code style

### PR DESCRIPTION
**Pull request by Pchelolo**
Most of the stuff fixed automatically, then I've looked over the changes and all seems to be fine.
The jscs config is the same as in parsoid, but we don't verify that comments are starting with an upper-case letter, because we have several places with commented-out code that could be useful for debugging.

Bug: https://phabricator.wikimedia.org/T107950

### CODE REVIEW

**Comment by d00rman** [comment]
I really really have a problem with trailing commas. I find them too confusing and I think they hurt readability.

**Comment by Pchelolo** [comment]
We could then use "disallowTrailingComma": true and remove all of them.

File: mods/action.js [hunk 1]
```diff
@@ -10,75 +10,75 @@ var Template = require('../lib/reqTemplate');
...
 var errDefs = {
-    '400': { status: 400, type: 'invalid_request' },
-    '401': { status: 401, type: 'unauthorized' },
-    '403': { status: 403, type: 'access_denied#edit' },
-    '409': { status: 409, type: 'conflict' },
-    '413': { status: 413, type: 'too_large' },
-    '429': { status: 429, type: 'rate_exceeded' },
-    '500': { status: 500, type: 'server_error' },
-    '501': { status: 501, type: 'not_supported' }
+    400: { status: 400, type: 'invalid_request'},
```
Comment (d00rman) [inline mods/action.js]: Why? If there is a space after the opening brace, I'd expect there to be one at the end as well (symmetry, OCD and all) :)

**Comment by d00rman** [comment]
> We could then use "disallowTrailingComma": true and remove all of them.

Thanks for that! Looks much nicer now. But, still, some bike-shedding comments... Code face-lifting is never as easy as it seems.

Also, this is a good opportunity to get rid of all of those ugly `console.log()` calls that are commented out. Thoughts?

**Comment by d00rman** [comment]
@Pchelolo mind rebasing on top of master so we have the io.js tests passing?

**Comment by Pchelolo** [comment]
@d00rman Updated, rebased, removed commented-out code

**Comment by d00rman** [comment]
Looks nice now, thnx @Pchelolo !

Bike-shed: what's the exact benefit of Capitalised comments? Makes sense if you have `/* */` blocks that explain stuff, but capitalising couple-of-word comments seems a bit excessive.

**Comment by Pchelolo** [comment]
@d00rman not sure what's the reason for comment capitalisation. We could switch off that check.

**Comment by gwicke** [comment]
I like how trailing commas simplify editing and diffing. When adding a new array entry or property, only a single line needs to be touched, and only a single line shows up dirty in the diff.

Given that @d00rman feels strongly about not using them, could we perhaps instate an 'anything goes' policy wrt to trailing commas?

**Comment by Pchelolo** [comment]
@gwicke ok, if @d00rman wouldn't be too upset about it)

**Comment by gwicke** [comment]
I'm also :+1: on ignoring comment capitalization.

**Comment by eevans** [comment]
@d00rman 
> I really really have a problem with trailing commas. I find them too confusing and I think they hurt readability.

I'm a little surprised that this doesn't appeal to your need for symmetry (because it does mine :)).  I'd also echo @gwicke's sentiment that from a practical standpoint, it creates a smaller more readable diff later if the structure is extended.

That said, I don't know that I'd reject a patch during code-review because it did *not* have a trailing comma, so I'm in favor of having the code-checker ignore one.

**Comment by eevans** [comment]
> not sure what's the reason for comment capitalisation. We could switch off that check.

Personally, I think you *should* use capitalization (and punctuation) when formatting comments (for the same reasons that you would in documentation, correspondence, etc).  And I say this with the full knowledge that if we audited my changesets, it'd likely paint me a hypocrite. :)

That said, I can't see myself pushing back on a change during code-review because of this, so I agree that the code checker should ignore it.

But I'd leave in place the capitalization changes that are already in this PR, because they make the code better.

**Comment by Pchelolo** [comment]
Updated the PR - we now ignore comment issues and trailing commas

**Comment by eevans** [comment]
:+1: from me; Thanks @Pchelolo!

**Pull request merged**